Gopkg.toml

```diff
@@ -124,3 +124,7 @@ ignored = [
 [[constraint]]
   branch = "master"
   name = "github.com/hashicorp/go-version"
+
+[[override]]
+  name = "github.com/Masterminds/sprig"
+  version = "v2.16.0"
```

Comment:
Why is this added? It doesn't seem to change the lockfile

Reply:
the first time i ran update it changed the sprig version back to `2.11.0`

Reply:
And sprig was already updated to v2.16.0 in #737

Reply:
That's odd...

Reply:
@Rob0h why `override` then, and not `constraint`?

Overrides are something to be avoided when possible